rcon/player_history.py

```diff
@@ -497,15 +505,15 @@ def remove_flag(
     return player, old_flag
 
 
-def get_player_messages(player_id: str) -> list[PlayerActionType]:
+def get_player_messages(player_id: str) -> list[PlayerActionState]:
     with enter_session() as sess:
         player = sess.query(PlayerID).filter_by(player_id=player_id).one_or_none()
-        actions: list[PlayerActionType] = []
+        actions: list[PlayerActionState] = []
```

Comment:
Minor thing but this should be `PlayerActionType`, same for the function's return type